diff --git a/phoenix++-1.0/Defines.mk b/phoenix++-1.0/Defines.mk
index ab056ba..4a200e5 100644
--- a/phoenix++-1.0/Defines.mk
+++ b/phoenix++-1.0/Defines.mk
@@ -38,8 +38,8 @@ ifeq ($(OSTYPE),Linux)
 OS = -D_LINUX_
 DEBUG = -g
 #NUMA = -DNUMA_SUPPORT
-CFLAGS = $(DEBUG) -Wall -O3 $(OS) $(NUMA) -DMMAP_POPULATE -fstrict-aliasing -Wstrict-aliasing 
-LIBS = -lpthread -lrt
+CFLAGS = $(DEBUG) -Wall -O3 $(OS) $(NUMA) -DMMAP_POPULATE -fstrict-aliasing -Wstrict-aliasing -fpermissive 
+LIBS = -pthread -lrt
 endif
 
 ifeq ($(OSTYPE),SunOS)
